server/dataset_context.h

```diff
@@ -109,12 +109,16 @@ struct SqlExpressionDatasetContext: public SqlExpressionMldbContext {
     {
         return RowContext(row, params);
     }
+
+    virtual Utf8String 
+    doResolveTableName(const Utf8String & fullVariableName, Utf8String &tableName) const;
```

Comment:
Well, both as it were. We have a variable name that may or may not contain a table name.